docs/source/apidoc/simulation.rst

```diff
@@ -5,9 +5,14 @@ Classical Simulation
 Since version 0.6.0, all simulation classes (previously under the ``pulser.simulation`` module)
 are in the ``pulser-simulation`` extension and should be imported from ``pulser_simulation``.
 
-Simulation
+QutipEmulator
 ----------------------
 
+:class:`QutipEmulator` is the class to simulate :class:`SequenceSamples`, that are samples of a :class:`Sequence`.
+It is possible to simulate directly a :class:`Sequence` object by using the class method 
+`SequenceSamples.from_sequence`. Since version 0.14.0, the :class:`Simulation` class is deprecated 
+for :class:`QutipEmulator`.
+
 .. automodule:: pulser_simulation.simulation
```

Comment:
We need to modify the docstring of the `pulser_simulation.simulation` module, as it right now is mentioning the `Simulation` class. Also, we should add a `Note:` section in the `Simulation` docstring saying it is deprecated and to use `QutipEmulator.from_sequence()` instead.